static/app/views/alerts/rules/metric/edit.spec.jsx

```diff
@@ -53,7 +53,8 @@ describe('MetricRulesEdit', function () {
     });
   });
 
-  it('renders and edits trigger', async function () {
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('renders and edits trigger', async function () {
```

Comment:
This timed out, and as per the announcement in the frontend channel, I'm disabling this for now. If the pipeline goes green, I'll post this update in the frontend channel.